packages/tangle/messagefactory_test.go

```diff
@@ -358,7 +363,11 @@ func checkReferences(t *testing.T, tangle *Tangle, parents MessageIDs, expectedR
 	}
 	require.NoError(t, err)
 
-	assert.Equal(t, parents, actualReferences[StrongParentType])
+	if expectedStrongParents, ok := expectedReferences[StrongParentType]; ok {
```

Comment:
Isn't this already covered by the loop over all parents type on line 372?

Reply:
I made the way we call the `checkreferences` function explicit. So we can handle strong parents the same as the others.